.circleci/config.yml

```diff
@@ -140,7 +140,7 @@ jobs:
       - run: yarn install
       - run: yarn pack:macos
       - run: yarn upload:macos
-      - run: ./scripts/promote-stable-rc-mac
+      # - run: ./scripts/promote-stable-rc-mac
```

Comment:
I am thinking i might enable this and make sure the channel is not `stable-rc` in `scrtips/promote-stable-rc-mac`. That way I could download the installer from the channel and try an install from scratch. Although, maybe that's not necessary? I could probably download the pkg from aws.

Reply:
yeah, the `oclif:pack` output should give you the AWS locations that you can download from.  

If you wanted to test what your command does when the user changes the channel (ex: `sfdx update stable-rc` when they were on stable).  For that, you'd probably want your own channel in each of the `promote-stable-rc-*` files

Reply:
If you need to release to a channel you can do the opposite of what this commit shows https://github.com/salesforcecli/sfdx-cli/pull/329/commits/ff54cc8d979252c58aab0c82960030077b325a57.
This is the commit where I removed all the stuff for testing releasing installers to a `dev` channel for sfdx/sf bundling.
I recommend expanding the `.circleci/config.yml` entirely so you can see the whole thing, not just the changes.


Notice I left the approval step because once you commit the builds _will_ kick off automatically. This let me double check everything before anything was actually released or uploaded to AWS. 
I also had the `release-management/release-package` step commented out so I could skip npm/github releases and go straight to installers/tarballs, otherwise the SF-CLI-BOT will make "release" commits to your branch, not critical, just annoying if you don't need the npm/github release for testing.
If you do end up releasing stuff to `npm` you'll probs want to [delete](https://docs.npmjs.com/cli/v8/commands/npm-dist-tag) w/e tag you use and [deprecate](https://docs.npmjs.com/cli/v8/commands/npm-deprecate) the versions afterwards.